src/Components/Timer/Timer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Timer from 'react-compound-timer';
+import style from './Timer.module.scss';
+
+// const COLOR_CODES = {
+//     info: {
+//         color: '#F4D2D0',
+//     }
+// }
+
+// let remainingPathColor = COLOR_CODES.info.color
+
+const CountdownTimer = (props) => {
+    // const calculateTimeFraction = () => {
+    //     let timeLeft = this.context.value.m * 60 + this.context.value * 1000
```

Comment:
So impressed with what you were able to get working. Maybe we can all look into this piece together later & see if we can figure it out!

Reply:
Totes down to help troubleshoot this. Twould be a lovely feature if we can get it working!